internal/nodes/avi_model_tls_passthrough.go

```diff
@@ -97,10 +97,14 @@ func (o *AviObjectGraph) BuildGraphForPassthrough(svclist []IngressHostPathSvc,
 	}
 	dsNode := datascriptList[0]
 	var infrasettingName string
+	var infraSettingNameWithSuffix string
 	var pgName string
 	if infraSetting != nil {
 		infrasettingName = infraSetting.Name
+		infraSettingNameWithSuffix = infrasettingName + "-"
 	}
+	//Replace AVIINFRA with infrasettingname if present
+	dsNode.Script = strings.Replace(dsNode.Script, "AVIINFRA", infraSettingNameWithSuffix, 1)
```

Comment:
isnt it better to just append infrasetting name rather than replacing it ?

Reply:
@apalsule: `PoolGroup` name text is in the middle of datascript text (dsNode.Script). So replacing the exact value with avi-infrasetting placeholder is the way to go ahead.